misk/src/main/kotlin/misk/hibernate/Session.kt

```diff
@@ -1,11 +1,20 @@
 package misk.hibernate
 
+import java.sql.Connection
 import kotlin.reflect.KClass
 
 interface Session {
   val hibernateSession: org.hibernate.Session
   fun <T : DbEntity<T>> save(entity: T): Id<T>
   fun <T : DbEntity<T>> load(id: Id<T>, type: KClass<T>): T
+  fun shards(): List<Shard>
+  fun <T> target(shard: Shard, function: () -> T): T
+  fun <T> doWork(work: (Connection) -> T): T
+}
+
+data class Keyspace(val name: String)
```

Comment:
assert well-formed? Would be lousy to accidentally let whitespace or delimiters in

Reply:
done